hyperspy/misc/hyperspy_magics.py

```diff
@@ -26,10 +37,9 @@ def hyperspy(self, line):
 
         Positional arguments:
         ---------------------
-            toolkit
+            toolkit : {qt4, gtk, wx, tk, None}
```

Comment:
Shouldn't this be: {"qt4", "gtk", "wx", "tk", "None"}

It would also be good to remark that HyperSpy only fully supports qt4 and wx. "gtk" and "tk" do not have gui elements.